book/content/rdm/rdm.md

```diff
@@ -19,6 +19,7 @@ The following sections in this handbook provide useful context and complementary
 6. [Storage and backup](#Storage-backup)
     i. [Where to store data](#Where-store)
     ii. [Backups](#Backups)
+6. [Data organisation in spreadsheets](#Spreadsheets)
```

Comment:
I've fixed the merge conflict, but can you update these numbers so they're consecutive. (currently we have two 6s!)

Reply:
markdown doesn't care which numbers you use for numbered lists (it will display as 1, 2, 3,... either way 🔢 ) but, yes, this is sloppy - will fix 😃